images/server/install-packages.sh

```diff
@@ -128,7 +128,7 @@ case "${install_packages_from}" in
     ;;
     custom-repos)
         get_custom_repos "${install_custom_repos}"
-        package_selection=${package_selection:-custom}
+        package_selection=${package_selection:-custom-repos-devbuilds}
```

Comment:
No complaint with this change, but I had figured people (usually me) doing custom builds would also be setting the appropriate `--package-selection` value each time. So that's why I hadn't changed it previously.